Package.resolved

```diff
@@ -5,18 +5,18 @@
         "package": "FoundationExtensions",
         "repositoryURL": "https://github.com/teufelaudio/FoundationExtensions",
         "state": {
-          "branch": null,
-          "revision": "c33815c48b003626e8b3e0955e15b0c6594f2573",
-          "version": "0.1.1"
+          "branch": "NewPromise",
+          "revision": "580710be8e113fbb398a5ec95ee242c7af4a4415",
```

Comment:
When merging https://github.com/teufelaudio/FoundationExtensions/pull/20 we can go back to versioned dependency